polaris.shopify.com/content/components/text.md

```diff
@@ -22,98 +22,26 @@ status:
+      Use to give text a range of font weights.
+  - fileName: text-color.tsx
+    title: Color
     description: >-
-      Use to create a 2xl heading text.
-  - fileName: text-heading-3xl.tsx
-    title: Heading 3xl
-    description: >-
-      Use to create a 3xl heading text.
-  - fileName: text-heading-4xl.tsx
-    title: Heading 4xl
-    description: >-
-      Use to create a 4xl heading text.
-  - fileName: text-body-small.tsx
-    title: Body small
-    description: >-
-      Use to create a small body text.
-  - fileName: text-body-medium.tsx
-    title: Body medium
-    description: >-
-      Use to create a medium body text.
-  - fileName: text-body-large.tsx
-    title: Body large
-    description: >-
-      Use to create a large body text.
-  - fileName: text-with-align-inherit.tsx
-    title: With align inherit
-    description: >-
-      Use to inherit parent alignment.
-  - fileName: text-with-align-start.tsx
-    title: With align start
-    description: >-
-      Use to align text at start of horizontal line.
-  - fileName: text-with-align-center.tsx
-    title: With align center
-    description: >-
-      Use to align text at center of horizontal line.
-  - fileName: text-with-align-end.tsx
-    title: With align end
-    description: >-
-      Use to align text at end of horizontal line.
-  - fileName: text-with-align-justify.tsx
-    title: With align justify
-    description: >-
-      Use to align text on a vertical line.
-  - fileName: text-with-font-weight-regular.tsx
-    title: With font weight regular
-    description: >-
-      Use to give text a regular font weight.
-  - fileName: text-with-font-weight-medium.tsx
-    title: With font weight medium
-    description: >-
-      Use to give text a medium font weight.
-  - fileName: text-with-font-weight-semibold.tsx
-    title: With font weight semibold
-    description: >-
-      Use to give text a semi-bold font weight.
-  - fileName: text-with-font-weight-bold.tsx
-    title: With font weight bold
-    description: >-
-      Use to give text a bold font weight.
-  - fileName: text-with-color-success.tsx
-    title: With color success
-    description: >-
-      Use in combination with a symbol showing an increasing value to indicate an upward trend.
-  - fileName: text-with-color-critical.tsx
-    title: With color critical
-    description: >-
-      Use in combination with a symbol showing a decreasing value to indicate a downward trend.
-  - fileName: text-with-color-warning.tsx
-    title: With color warning
-    description: >-
-      Use to denote something that needs attention, or that merchants need to take action on.
-  - fileName: text-with-color-subdued.tsx
-    title: With color subdued
-    description: >-
-      Use to de-emphasize a piece of text that is less important to merchants than other nearby text. May also be used to indicate when normal content is absent, for example, “No supplier listed”. Don’t use only for aesthetic effect.
```

Comment:
I did end up removing these usage guidelines for colors, not sure if there's a good place to include them? Would it make sense to maybe include them in the examples directly? They end up in the codepens then.
<img width="783" alt="Screen Shot 2022-07-28 at 9 46 18 AM" src="https://user-images.githubusercontent.com/3474483/181593231-9befc134-806d-40e9-b541-1e41431a332f.png">

Reply:
It could replace the `Orders increased` text?

Reply:
good idea!💡

<img width="773" alt="Screen Shot 2022-07-28 at 10 25 08 AM" src="https://user-images.githubusercontent.com/3474483/181599790-7c396214-ebb0-4a3c-8ab1-2959efb915aa.png">

Reply:
I like the descriptions inline here. Feels like pretty good way to display the info. We can always iterate on this too if needed.